prefect_dask/utils.py

```diff
@@ -0,0 +1,167 @@
+"""
+Utils to use alongside prefect-dask.
+"""
+
+from contextlib import asynccontextmanager, contextmanager
+from datetime import timedelta
+from typing import Any, Dict, Optional, Union
+
+from distributed import Client, get_client
+from prefect.context import FlowRunContext, TaskRunContext
+
+from prefect_dask.exceptions import ImproperClientError
+
+
+def _populate_client_kwargs(
```

Comment:
I'd call this `_generate_client_kwargs` — populate hints to me that we're setting attributes on an object which is not the case here.